### DISCUSSION TITLE
Need help on bandwidth usage

### DISCUSSION THREAD

**Discussion by deepthydon** [vercel/vercel#7890] [Help]
Hi All,

I am currently on the Pro plan, and although the bandwidth indicates no limit, I saw on the docs that there is a 1TB limit (fair use policy). Currently, every day my usage is on an average 100GB.

I am a bit worried because i already crossed 1TB limit and paying additional amount.

What is the best way to find out what is eating up all that bandwidth? is there any way to optimize the bandwidth usage.

I eagerly await your answer & appreciate your quick help on this.

My technical stack prismic + nextjs.

Best regards,
Deepthy.

**Comment by skllcrn** [comment]
Hi @deepthydon, thank you for your message! If you navigate to `vercel.com/dashboard/[team-slug]/usage`, replacing the `[team-slug]` with your the name/slug of your team on the Pro plan, you will see a Bandwidth section under Networking that also indicates your standing in relation to the fair use policy. You'll also see this information in the Billing section of your Team Settings. 

We're working on new ways to identify what usage is contributing the most to spend and I'd love to run those by you to see if they would help in your case. If you'd be up for that or continue having issues, please [contact us](https://vercel.com/support/request) and include a link to this issue, and we'll be in touch!

**Reply by deepthydon** [reply]
hi @skllcrn : Thanks for the reply. Unfortunately usage dashboard not giving any detailed breakup and without details it's very difficult to troubleshoot.

At the moment i am clueless why the bandwidth usage is high. Detailed report would definitely help here.

Also, in general what could be the cause of high usage. Our traffic is high on the website. On an average I could expect 2k to 3 k users daily visit for my website. Does this create any issue on the usage ?

**Reply by skllcrn** [reply]
Generally speaking the number of visits shouldn't be the primary concern if your caching and static assets are optimized to avoid doing extra work for additional requests. You may also want to set up a [Log Drain](https://vercel.com/integrations#logging) for your project. 

It'd be great to chat more and see how our future improvements can help you achieve this type of breakdown, just reach out via the link in my initial reply!

**Reply by deepthydon** [reply]
Thank you very much for the support @skllcrn . Since i am new to this technical stack would be great if you can provide some guidance to optimize the bandwidth consumed by next image

**Reply by skllcrn** [reply]
Happy to help @deepthydon! In addition to the [Image Optimization](https://vercel.com/docs/concepts/image-optimization) docs, you may want to reference the [Production Checklist](https://vercel.com/docs/concepts/production-checklist), and also our [guide on lowering usage](https://vercel.com/guides/lower-usage#image-optimization-usage).

**Reply by deepthydon** [reply]
@skllcrn : Thanks for the detailed response. One last question, I have two diffrent types of images in my application. 
1. Static images -> url format like /_next/image?url=
2. Another set of images are coming from prismic. -> https://images.prismic.io

Ideally vercel shouldn't calculate the image are loading from prismic for bandwidth right ? Or my understanding is totally wrong ?

**Reply by skllcrn** [reply]
Images served by us via `next/image` count toward your bandwidth and image use. If you'd like to serve them directly from a third party you'll want to opt for the [unoptimized flag](https://nextjs.org/docs/api-reference/next/image#unoptimized) (`<Image src={img} unoptimized />`) or use the `<img />` tag.

**Reply by deepthydon** [reply]
@skllcrn  : Once again thank you very much for the continuous and quick support. I believe the situation improved now after we put few optimisation (especially image). Let us monitor few more days and i will keep you posted the final status
